src/ray/gcs/gcs_server/gcs_kv_manager.cc

```diff
@@ -54,15 +54,15 @@ void GcsInternalKVManager::HandleInternalKVMultiGet(
       return;
     }
   }
-  auto callback =
-      [reply, send_reply_callback](std::unordered_map<std::string, std::string> results) {
-        for (auto &result : results) {
-          auto entry = reply->add_results();
-          entry->set_key(result.first);
-          entry->set_value(result.second);
-        }
-        GCS_RPC_SEND_REPLY(send_reply_callback, reply, Status::OK());
-      };
+  auto callback = [reply, send_reply_callback](
+                      absl::flat_hash_map<std::string, std::string> results) {
+    for (auto &&result : std::move(results)) {
```

Comment:
I still think `&&` and `std::move` here reduce readability, but no strong opinion